src/libraries/System.Runtime.CompilerServices.VisualC/src/System.Runtime.CompilerServices.VisualC.TypeForwards.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should we be creating two of these files instead of just sharing one?

Reply:
IIRC shouldn’t need to include this file in the src project, see my comment below.

Reply:
I was able to follow Santi's suggestion and remove the file in the src project.